Juice that makes you go boom

Velken opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
Added new explosive reaction when mixing "Juice that makes you go Weh" and "Juice that makes you go Hew"

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Just like combining matter and anti-matter can cause devastating effects, letting pure essence of Lizard and Drazil mix will lead to unforseen consequences.

The explosion has a considerable radius and damage, but since Drazil plushies are so rare, I believe it should be fine.

## Technical details
<!-- Summary of code changes for easier review. -->
Added new entry on chemicals.yml

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

NOTE: video is slightly outdated, there is no longer spacing damage!!

https://github.com/user-attachments/assets/441e3669-469a-4d4d-ab45-f48e3c7d8d6d


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

None.

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- add: Mixing opposite types of lizard juices now has explosive consequences.

lzk228 commented:
make it create wehularity :trollface: (joking ofc don't do that)

ArtisticRoomba commented on Resources/Prototypes/Recipes/Reactions/chemicals.yml:
```diff
@@ -516,3 +516,26 @@
   products:
     Tazinide: 1
 
+- type: reaction
+  id: WehHewExplosion
+  impact: High
+  priority: 20
+  reactants:
+    JuiceThatMakesYouWeh:
+      amount: 1
+    JuiceThatMakesYouHew:
+      amount: 1
+  effects:
+    - !type:ExplosionReactionEffect
+      explosionType: Default
+      maxIntensity: 100
+      intensityPerUnit: 1
+      intensitySlope: 1
+      maxTotalIntensity: 125
+    - !type:ExplosionReactionEffect
+      explosionType: Radioactive
+      maxIntensity: 100
+      intensityPerUnit: 1
+      intensitySlope: 1
+      maxTotalIntensity: 125
+
```
this should probably be moved to `Resources/Prototypes/Reagents/fun.yml` where the weh and hew chems are listed

Velken replied:
I though of putting it on fun, but since it was a reaction, i put it chemicals (with the other chemical reactions, like potassium, explosion)

Mixelz commented:
I think this has a very high chance of being meta gamed to hell and causing hew plushies to be soley used for this purpose whenever found by an antag that is in the know and be a piece of non-official contraband due to this one interaction alone.

Velken commented:
moved explosive juice to fun yml; 
fixed ExplosionReactionEffect.cs not having TileBreakScale paramete;
made Drazil plushie major contraband (they are evil!!!!!)

pheenty commented:
Juice that makes your head explode

slarticodefast commented on Content.Server/EntityEffects/Effects/ExplosionReactionEffect.cs:
```diff
@@ -50,6 +50,14 @@ public sealed partial class ExplosionReactionEffect : EntityEffect
+    ///     explosives that don't space tiles, without having to create a new explosion-type prototype.
+    /// </summary>
+    [DataField]
+    [JsonIgnore]
```
```suggestion
```
We don't even use Json, so this should be removed. Same for the ones above.

ScarKy0 commented on Resources/Prototypes/Recipes/Reactions/chemicals.yml:
```diff
@@ -516,3 +516,4 @@
   products:
     Tazinide: 1
 
+
```
whitespace :(

ScarKy0 commented on Resources/Prototypes/Entities/Objects/Fun/toys.yml:
```diff
@@ -437,7 +437,7 @@
         offset: "0, 0.03"
 
 - type: entity
-  parent: PlushieLizard
+  parent: [ PlushieLizard, BaseMajorContraband ]
   id: PlushieLizardInversed #Hew!
```
Personally im against it being major contraband.
You can already make big explosions with water and potassium, which are WAY more easily available and neither are contraband.

ScarKy0 reviewed (changes requested):
Get reviewed nerd

ScarKy0 reviewed (approved):
Looks good to me

FairlySadPanda commented:
At this point we're about one PR away from this becoming the Wizden equivalent to the old Chaos Dunk event from '13.

Velken commented:
> At this point we're about one PR away from this becoming the Wizden equivalent to the old Chaos Dunk event from '13.

Considering how rare Drazils are, it is literally easier to make potassium bombs of same (or even greater) damage than this.

ArtisticRoomba reviewed (approved):
LGTM

Velken commented:
just had to do a small ident fix before aeshus comes here 🤣

Velken commented:
Two explostions at once makes stuff be pushed twice, now is just 1 boom
![2025-01-29-22 48 35-Content Client](https://github.com/user-attachments/assets/45352eab-2a27-4f7e-9e12-6b465b21c497)
![2025-01-29-22 48 55-Content Client](https://github.com/user-attachments/assets/6301d693-a186-45d5-afdb-53f45aead1ad)

slarticodefast commented:
Being at the epicenter of the explosion can cause quite a bit of damage, but considering they are rare to obtain and you can do the same with water and potassium as well it might be fine. We can give it a try and nerf the explosion if neccessary.
Thank you for your contribution!

notquitehadouken commented:
juice that makes your head explode
